include/radix/core/gl/OpenGL.hpp

```diff
@@ -1,6 +1,10 @@
 #ifndef RADIX_CORE_GL_OPENGL_HPP
 #define RADIX_CORE_GL_OPENGL_HPP
 
+#ifdef _WIN32
+#define RADIX_GL_USE_GLAD
```

Comment:
CMake is relevant here since it generates the VS project files. And you are forcing GLAD on Windows since `_WIN32` is defined by [virtually all compilers when running on Windows](http://nadeausoftware.com/articles/2012/01/c_c_tip_how_use_compiler_predefined_macros_detect_operating_system#WindowsCygwinnonPOSIXandMinGW).